Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG base_image=python:3.9-alpine@sha256:709505ac2ed5824430abb76db8cd24c45415aa1f267e133546977e0b18241d3e
```

Comment:
Do you need the extra - here?

Reply:
Which is the extra? Between `3.9` and `alpine`? I think it's just part of the tag?